docs-src/multiple-ns.md

```diff
@@ -0,0 +1,37 @@
+# Routes in multiple namespaces
+
+With the Service APIs, a single Gateway can target routes across multiple
+namespaces. 
+
+This guide assumes that you have installed Service APIs CRDs and a conformant
+controller.
+
+In the following example:
+
+- `acme-lb` GatewayClass: The GatewayClass responsible for satisfying Gateway
+  and Route resources.
+- `multi-ns-gateaway` Gateway: The Gateway is configured with a single listener
+  on port 80 which selects routes from **all namespaces** which have the label
+  `product: baz`. Notice how the `routes.namespaces.from` field in the listener
+  is set to `All`.
+- `service-apis-example-ns1` and `service-apis-example-ns2` Namespaces: These
+  are two namespaces in which we route resources are installed.
```

Comment:
s/These are two namespaces in which we route resources are installed/These are the namespaces in which route resources are instantiated.